web/filament-js/jsbindings.cpp

```diff
@@ -772,8 +776,8 @@ class_<VertexBuilder>("VertexBuffer$Builder")
             VertexAttribute attr,
             uint8_t bufferIndex,
             VertexBuffer::AttributeType attrType,
-            uint8_t byteOffset,
-            uint8_t byteStride), {
+            size_t byteOffset,
+            size_t byteStride), {
```

Comment:
Nice catch but shouldn't offset be `uint32_t` and stride be `uint8_t`?

Reply:
@griffin2000 are you able to implement this change?  If not we can do it on our end.